packages/serverpod/lib/src/database/database.dart

```diff
@@ -70,6 +70,40 @@ class Database {
     );
   }
 
+  /// Find a list of [Distinct Column]s from a table, using the provided [where]
```

Comment:
Wouldn't this be better as an option to the `find` method, rather than adding a whole new method?

E.g

```
var companies = await Company.find(
  tCompany,
  where: (t) => t.id < 100,
  limit 50,
  distinct: true,
);
```

Reply:
can't able to add to find method. because return value varies
for Distinct values, only the mentioned column with their Distinct value will be returned

```
final result = SampleClass.findDistinctValue(session,
        isDistinct: true, columns: [SampleClass.t.name], returnAsList: true);
result => [['Anand'],['Subbu']];
```
```
final result = SampleClass.findDistinctValue(session,
        isDistinct: true, columns: [SampleClass.t.name,SampleClass.t.age], returnAsList: true);
result => [['Anand','21'],['Subbu','23']];
```

```
final result = SampleClass.findDistinctValue(session,
        isDistinct: true, columns: [SampleClass.t.name]);
result => [{'name': 'Anand'}];
```
```
final result = SampleClass.findDistinctValue(session,
        isDistinct: true, columns: [SampleClass.t.name,SampleClass.t.age]);
result => [{'name': 'Anand','age':23}];
```